core/src/plan/expr/function.rs

```diff
@@ -6,13 +6,14 @@ use {
 impl Function {
     pub fn as_exprs(&self) -> impl Iterator<Item = &Expr> {
         #[derive(iter_enum::Iterator)]
-        enum Exprs<I0, I1, I2, I3, I4, I5> {
+        enum Exprs<I0, I1, I2, I3, I4, I5, I6> {
             Empty(I0),
             Single(I1),
             Double(I2),
             Triple(I3),
             VariableArgs(I4),
             VariableArgsWithSingle(I5),
+            Quadruple(I6),
```

Comment:
Is it correct to create a `Quadruple` instead of a `VariableArgs`? @panarch

Reply:
for now, both can be accepted so this decision which adds Quadruple looks also good.

nit. it would be better to add Quadruple in between Triple and VariableArgs rather than adding it to the last.